Sprout/View Controllers/HomeViewController.m

```diff
@@ -7,26 +7,69 @@
 
 #import "HomeViewController.h"
 #import "Parse/Parse.h"
+#import "TaskCell.h"
+#import "Task.h"
 
-@interface HomeViewController ()
+@interface HomeViewController () <UITableViewDelegate, UITableViewDataSource>
+
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (nonatomic, strong) PFUser *user;
+@property (nonatomic, strong) NSMutableArray *arrayOfTasks;
 
 @end
 
 @implementation HomeViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    
+    self.user = PFUser.currentUser;
+    
+    [self queryTasks:20];
 }
 
-/*
-#pragma mark - Navigation
+- (void) queryTasks:(int) numPosts {
+    //TODO: add refresh control. Later on: sort by category?
+    PFQuery *query = [PFQuery queryWithClassName:@"Task"];
+    //[query orderByDescending:@"createdAt"];
+    [query includeKey:@"author"];
+    [query includeKey:@"taskName"];
+    [query includeKey:@"type"];
+    [query includeKey:@"completed"];
+    
+    query.limit = numPosts;
+    
+    [query whereKey:@"author" equalTo:self.user];
+
+    [query findObjectsInBackgroundWithBlock:^(NSArray *arrayOfTasks, NSError *error) {
+        if (arrayOfTasks != nil) {
+            self.arrayOfTasks = (NSMutableArray *)arrayOfTasks;
+            [self.tableView reloadData];
+            //self.morePostsLoading = false;
```

Comment:
It's tempting to keep stuff around but it's easy to go back and get it from history.